alt-tab-macos/ui/PreferencesPanel.swift

```diff
@@ -2,6 +2,19 @@ import Cocoa
 import Foundation
 
 class PreferencesPanel: NSPanel, NSWindowDelegate {
+    let ACCESSIBILITY_WARNING_TEXT = """
+Before using this app, you need to give permission in System Preferences > Security & Privacy > Privacy > Accessibility.
+Please authorize and re-launch.
+
+See https://help.rescuetime.com/article/59-how-do-i-enable-accessibility-permissions-on-mac-osx
+"""
+    let SCREENSHOT_WARNING_TEXT = """
+Before using this app, you need to give permission in System Preferences > Security & Privacy > Privacy > Screen Recording.
+Please authorize and re-launch.
+
+See https://dropshare.zendesk.com/hc/en-us/articles/360033453434-Enabling-Screen-Recording-Permission-on-macOS-Catalina-10-15-
+"""
+
```

Comment:
I like the idea to move the console log to a more user-friendly UI, but I think the preference panel is not the right place. It should be about setting preferences. I think it would be better to have a dedicated window that shows if you start the app and didn't check the permission checkboxes. That new window would also need to either have a "restart the app" button, or listen to some OS event when the checkboxes are checked, so the window closes itself automatically

Reply:
The idea here was that the first thing you do when installing the app is inevitably opening the preferences, and you'd see the warning(s) there. 

I don't really necessarily have the Swift-fu for that dedicated window, so this was simpler :)

Reply:
I think that merging this PR as-is could be considered a regression. I remember when #33 was introduced to fix the issue on Catalina, people were saying that they were not getting the permission prompt every time. We added it at startup, so that if it's not checked, it stops, hopefully the user gets the OS prompt, and this can loop until the user has checked the box.

This PR as-is, puts permissions as a background thing for the user to do. It means that they can have the app running, not working, and potentially confused as to what's happening. I think it's not necessarily a better UX.

I think to truly improve on the UX, we need some dedicated onboarding UI to instruct the users to check the boxes.